### PR TITLE
Easily create lunch poll

### PR DESCRIPTION
**Pull request by jonsongoffwhite**

### CODE REVIEW

File: scripts/lunch.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = (robot) ->
+    robot.hear /What are we having for lunch?/i, (res) ->
+        res.send("/poll Where should we go for lunch? :gbk: :pizza: :spaghetti: :curry: :sushi: :taco: :burrito: :ramen: :chicken:")
```
Comment (browniebroke) [inline scripts/lunch.coffee:3]: @jhg14 Would be good to have it documented in the robot help maybe? I let you have the wow effect tomorrow first ;-)

Comment (jonsongoffwhite) [reply scripts/lunch.coffee:3]: I'm on holiday tomorrow 😢 I'll witness the magic remotely!

Comment (nanorepublica) [reply scripts/lunch.coffee:3]: The regex could have been more flexible 😛  it won't catch 'Where are we going for lunch?'

Comment (browniebroke) [reply scripts/lunch.coffee:3]: NOT IN THE MVP 😂

**Pull request merged**